TrustWalletCore.podspec

```diff
@@ -126,12 +128,13 @@ Pod::Spec.new do |s|
       "#{protobuf_source_dir}/src/**/*.{h,inc}",
       "#{include_dir}/nlohmann/**/*.hpp",
       'src/proto/*.proto'
+
     ss.xcconfig = {
       'HEADER_SEARCH_PATHS' => '$(inherited) ' \
         '${PODS_ROOT}/TrustWalletCore/src ' \
         '${PODS_ROOT}/TrustWalletCore/trezor-crypto/src',
       'SYSTEM_HEADER_SEARCH_PATHS' => '$(inherited) ' \
-        '/usr/local/include' \
+        '/usr/local/include ' \
```

Comment:
```suggestion
        '/usr/local/include' \
```

Reply:
This is not a typo, it must be separated by spaces